apps/explorer/src/app/hooks/use-txs-data.ts

```diff
@@ -29,11 +29,13 @@ export const getTxsDataUrl = ({ limit, filters }: IGetTxsDataUrl) => {
     url.searchParams.append('limit', limit);
   }
 
+  // Hacky fix for param as array
+  let urlAsString = url.toString();
```

Comment:
question (non-blocking): again, just out of curiosity, what do you get when you do `url.searchParams.append('filters', filters)` and then do `url.toSting()`?

Reply:
If you do that, you get `...transactions?limit=10&filters=filters%5Btx.submitter%5D%3Dbd6339d2428c79ac3bc9011771236d17bac92bcb1806423388d52fb440043aef`

What I lazily did not try is removing the `filters=` from the value and seeing if it was happy to append the `[tx.submitter=blah]` array item as is. I will try it out in a follow up.